content/blog/neutron-where-is-my-packet-2.md

```diff
@@ -0,0 +1,1082 @@
+---
+title: "OpenStack Networking - Where Is My Packet - tunnel tenant networks, centralized routers"
+date: 2022-01-21T20:32:52+01:00
+draft: true
+---
+
+## Intro
+
+This is first post about how to track packet flow in the OpenStack Neutron
+connectivity. I want to focus here on the packet flows to/from virtual machines
+in such scenario. If You look for the detailed description e.g. about how
+Neutron agents works together, please refer to the upstream [Neutron
+Documentation](https://docs.openstack.org/neutron/latest/admin/).  Everything
+what will be described in that post is related to the case when Neutron is used
+with the ``ML2`` core plugin and the ``openvswitch`` mechanism driver.
+Everything what will be shown in this article requires access to the OpenStack
+cluster nodes as root user. Access to the OpenStack API will is not enough to do
```

Comment:
I want to focus here on packet flows to/from virtual machines in a scenario with the `ML2` core plugin and the `openvswitch` mechanism driver. Some of the demonstrated steps and commands require access to the OpenStack API and others require root user privileges in the nodes of the OpenStack cluster. For a detailed description about how Neutron agents work together, please refer to the upstream [Neutron Documentation](https://docs.openstack.org/neutron/latest/admin/).

NOTE TO SLAWEK: Please note that I replaces ``ML2`` and ``openvswitch`` with `ML2` and `openvswitch`, assuming that you want to highlight these words as inline code. The double ticks are used in rst doucments, while single ticks are used in markdown